lib/reverse_sentence.rb

```diff
@@ -1,6 +1,54 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^3)
+# Space complexity: O(n) I believe since there is a second array created in memory to store/reverse each word 
```

Comment:
Yes since you're doing `.split` it creates a new array, and you create a temp array.

You were asked to reverse the words in place.